README.md

```diff
@@ -32,6 +32,10 @@ If you use Fresco to load images in your application, take a look at Datadog's [
 
 If you use Glide to load images in your application, take a look at Datadog's [dedicated library](dd-sdk-android-glide/README.md).
 
+### Jetpack Compose
+
+If you use Jetpack Compose to load images in your application, take a look at Datadog's [dedicated library](dd-sdk-android-compose/README.md).
```

Comment:
it seems here is a copy-paste mistake from the Glide integration above. Jetpack Compose is not a library for loading images, it is a [UI toolkit framework](https://developer.android.com/jetpack/compose). I suppose the part about loading images should be deleted:

```suggestion
If you use Jetpack Compose in your application, take a look at Datadog's [dedicated library](dd-sdk-android-compose/README.md).
```